wave_lang/kernel/wave/cache.py

```diff
@@ -137,6 +137,7 @@ def anonymize_constraints(input_constraints: list[Constraint]):
             constraint.wave_id = None
         else:
             continue
```

Comment:
Good find. Could you update the type signature of this function to include the return type?

Reply:
Thanks!  Fixed in the most recent update.